package.json

```diff
@@ -41,6 +42,7 @@
     "ejs": "^3.0.1",
     "express": "^4.17.1",
     "fast-levenshtein": "^2.0.6",
+    "install": "^0.13.0",
```

Comment:
What is this dependency?

Reply:
I have no idea. Removed it and it runs the same as before.